pkg/karmadactl/interpret/execute.go

```diff
@@ -80,15 +80,18 @@ func (o *Options) runExecute() error {
 		Replica:  int64(o.DesiredReplica),
 	}
 
-	interpreter := configurableinterpreter.NewConfigurableInterpreter(nil)
-	interpreter.LoadConfig(customizations)
+	configurableInterpreter, err := configurableinterpreter.NewConfigurableInterpreter(nil)
+	if err != nil {
+		return fmt.Errorf("failed to new a configurable interpreter: %v", err)
+	}
+	configurableInterpreter.LoadConfig(customizations)
```

Comment:
Oh, I get it, you want to use the `LoadConfig` function, that's the reason why `NewConfigurableInterpreter` can't just return a `framework.Interpreter` interface.

Reply:
yes.